inc/namespace.php

```diff
@@ -150,6 +150,15 @@ function require_auth() {
 		return;
 	}
 
+	$allowed = [
+		'/robots.txt',
+	];
+
+	if ( $_SERVER['REQUEST_URI'] && in_array( $_SERVER['REQUEST_URI'], $allowed, true ) ) {
```

Comment:
Same as the other PR, this conditional is only true for this specific URL, so having an `$allowed` array gives the impression it applies to more cases. Let's stick with a simple string comparison here.

Reply:
Could we leave it as an array but make it filterable? It's useful in other cases, and we have [an equivalent for hm-require-login](https://github.com/humanmade/hm-require-login/blob/master/inc/namespace.php#L44) (which we use for allowing eg OAuth through)